lib/pelemay.ex

```diff
@@ -126,11 +134,11 @@ defmodule Optimizer.Enum do
     func
   end
 
-  def call_nif(num, :chunk_every, module) do
-    quote do: unquote(module).chunk_every(unquote(num))
+  def call_nif(num, :chunk_every) do
+    quote do: Temp.chunk_every(unquote(num))
```

Comment:
I think `Temp` is not suitable name.
We should adopt an alternative name after `Module`.